en/docs/learn/configuring-saml2-web-single-sign-on.md

```diff
@@ -178,7 +178,11 @@ To configure SAML2 Web SSO:
     </tr>
     <tr class="even">
     <td><strong>Enable Signature Validation in Authentication Requests and Logout Requests</strong></td>
-    <td>Select <strong>Enable Signature Validation in Authentication Requests and Logout Requests</strong> if you need this functionality configured. This specifies whether the identity provider must validate the signature of the SAML2 authentication request and the SAML2 logout request that are sent by the service provider.</td>
+    <td>Select <strong>Enable Signature Validation in Authentication Requests and Logout Requests</strong> if you need this functionality configured. This specifies whether the identity provider must validate the signature of the SAML2 authentication request and the SAML2 logout request that are sent by the service provider.
+	<div class="admonition tip">
+    <p class="admonition-title">Tip</p>
+    <p>To find the configuration recommended to enable signature validation in Travelocity, refer [Testing with tenants](../../setup/configuring-a-sp-and-idp-using-configuration-files.md#testing-with-tenants).</p></div>
+    </td>
```

Comment:
(../../setup/configuring-a-sp-and-idp-using-configuration-files**.md**#testing-with-tenants)

please remove the ".md" from this link.

Reply:
Fixed https://github.com/wso2/docs-is/pull/1420/commits/b8e195ba0af0406afea36c86707e50190dd5705d